src/pages/NewChatPage.js

```diff
@@ -179,10 +181,15 @@ class NewChatPage extends Component {
      * @param {Object} option
      */
     createChat(option) {
-        Report.fetchOrCreateChatReport([
-            this.props.session.email,
-            option.login,
-        ]);
+        let newChat = {};
+        const formattedLogin = OptionsListUtils.addSMSDomainIfPhoneNumber(option.login).toLowerCase();
+        const chat = ReportUtils.getChatByParticipants([formattedLogin], this.props.reports);
+        if (!chat) {
+            newChat = ReportUtils.buildOptimisticChatReport([formattedLogin]);
```

Comment:
Notification preference should default to `CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS`.

Reply:
Why is this and should this be the case everywhere for all optimistic chats or just here?

Reply:
We set it on the backend [here](https://github.com/Expensify/Web-Expensify/blob/92c356b0d420f092410f6bbd92280ebf9e68b23d/lib/Report.php#L731) so it's best to match that in the optimistic data. I have [this PR](https://github.com/Expensify/App/pull/11653/files) open to fix it for other chat types.

Reply:
If you want to get this merged more quickly then don't worry about it and I can clean it up in that PR.